Jekyll/Compare_Data_File_Types/JSON.md

```diff
@@ -0,0 +1,52 @@
+# JSON
+[Official site](https://www.json.org/)
+
+JSON (JavaScript Object Notation) is a lightweight data-interchange format. Easy for humans to read and write, easy for machines to parse and generate.
+
+JSON is built on:
+- Collection of name/value pairs.
+- An ordered list of values.
+
+## JSON Formats
+### Object
+- An unordered set of name/value pairs.  
+![objectImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/object.png)  
+Starts with left brace, {, and ends with right brace, }.  
+For example:
+```
+{ Apple : good }
+```
+
+### Array
+- An ordered collection of values.  
+![arrayImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/array.png)  
+Starts with left bracket, [, and ends with right bracket, ].  
+For example:
+```
+[ red, green, blue ]
+```
+
+### Value
+- It can be a *string* in double quotes, or a *number*, or *true* or *false* or *null*, or an *object* or an *array*.  
+![valueImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/value.png)  
+
+Objects and arrays use values with that form.
+
+### String
+- A sequence of Unicode characters wrapped in double quotes  
+![stringImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/String.png)  
+For example:
+```
+"Deja Vu I've just been in this place before"
+```
+
+### Number
+- Usually use digit numbers  
+![numberImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/number.png)
+
+### Whitespace
+- Can be inserted between any pair of tokens excepting a few encoding deatails  
+![whitespaceImage](https://github.com/hyecheol123/KSEAWebDocuments/blob/Yongsang_Jekyll_DirectoryStructure/Jekyll/Compare_Data_File_Types/JSON/whitespace.png)
+
+
+## Use JSON on Jekyll
```

Comment:
Need example about JSON how to retrieve this data file on Jekyll (Maybe code sample?) and use data inside it

Reply:
I checked it is on the main file